styles/_grid.scss

```diff
@@ -0,0 +1,79 @@
+.grid,
```

Comment:
From the official Foundry system template and used by the item sheet template (though not very much).